### ISSUE TITLE
wrong transformation properties for EPSG 31253, 31252, 31251 

### ISSUE THREAD

**Issue by mdiener21** [jswhit/pyproj#1]
proper settings should be for EPSG 31253, 31252, 31251 +towgs84=577.326,90.129,463.919,5.137,1.474,5.297,2.4232

NOT +towgs84=682,-203,480,0,0,0,0

**Comment by micahcochran** [comment]
I'm using Proj "Rel. 4.8.0, released 6 March 2012."

When you call Proj4's `pj_get_def()` on one of those above mentioned projections, you get this:
```
+init=epsg:31253 +proj=tmerc +lat_0=0 +lon_0=34 +k=1 +x_0=0 +y_0=-5000000 +ellps=bessel +towgs84=682,-203,480,0,0,0,0 +pm=ferro +units=m +no_defs'
```

`pj_get_def()` is not exposed in pyproj.  This seems to be a problem with underlying [proj.4 library](https://github.com/OSGeo/proj.4) that pyproj is built upon.

Proj.4 version 4.9.2  was released in September 2015, which might have corrected your issue.

**Comment by micahcochran** [comment]
I just looked at [PROJ.4's epsg file](https://github.com/jswhit/pyproj/blob/master/lib/pyproj/data/epsg).  The current development code has the version that you report as problematic, so upgrading to PROJ.4 version 4.9.2 will not fix your problem.

I also poked into QGIS's srs.db.  It has the Proj4 string as the reported problem.  It imports projections from PROJ.4 and GDAL's data.

If I were you, I would try reporting this to [PROJ.4](https://github.com/OSGeo/proj.4).  This library can also be updated afterwards.
